src/Stratis.Bitcoin/Builder/Feature/FeatureRegistration.cs

```diff
@@ -49,6 +51,15 @@ public interface IFeatureRegistration
         /// <returns>This interface to allow fluent code.</returns>
         IFeatureRegistration DependOn<TImplementation>() where TImplementation : class, IFullNodeFeature;
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <typeparam name="TOneImplementation"></typeparam>
+        /// <typeparam name="TTwoImplementation"></typeparam>
+        /// <returns></returns>
+        IFeatureRegistration DependOn<TOneImplementation, TTwoImplementation>()
```

Comment:
we should have 2 overloads for DependOn - one with just 1 feature and 2nd with any amount of features (so if we can resolve at least 1 dependency out of those it's ok)

Reply:
This whole dependency will be easier managed if we have an abstract consensus feature (even if its empty) basically it will mean the node must have consensus.